LabGui.py

```diff
@@ -1139,7 +1139,9 @@ def stop_DTT(self):
 
```

Comment:
Before you implement a new feature, I would recommend that you tell me about it, as I can quickly tell you where to look :), or whether it has been implemented.